plugins/nanospell/plugin.js

```diff
@@ -421,6 +407,7 @@
 				var spellCheckSpan = elementPath.contains(isSpellCheckSpan);
 
 				if (spellCheckSpan) {
+					target = findNearestParentBlock(target);
```

Comment:
Ah.  I was going to ask about this.

I thought more about my earlier suggestion to fold `checkNow()` and `triggerSpelling()` together.  I was thinking we could enforce that a block was passed in, so as to avoid the problem you asked about in hipchat.  Then I see that you are enforcing it down here.  :)